rewrite-yaml/src/main/java/org/openrewrite/yaml/MergeYaml.java

```diff
@@ -121,6 +121,9 @@ public Validated<Object> validate() {
         return super.validate()
                 .and(Validated.test("yaml", "Must be valid YAML",
```

Comment:
```suggestion
                .and(Validated.required("key", key))
                .and(Validated.test("yaml", "Must be valid YAML",
```

I we want to go down this road, add this as well 😉